api/server/advisory/client_connect.go

```diff
@@ -37,9 +37,7 @@ func init() {
           Name: {{ .Client.Name }}
 {{- end }}
        Account: {{ .Client.Account }}
-{{- if .Client.Lang }}
-      Language: {{ .Client.Lang }} {{ .Client.Version }}
-{{- end }}
+       Library: {{ with .Client.Version }}version={{ . }} {{ end }}{{ with .Client.Lang }}language={{ . }}{{ else }}language-unknown{{ end }}
```

Comment:
Since we have `language=`, would `unknown` not be enough (instead of `language-unknown`)?

Reply:
I think 

```
Library Version: 1.2.3 Language: Go
```

```
Library Version: 1.2.3 Language: Unknown
```

the `=` seems a bit at odds with how the rest of the text looks

Or maybe just

```
Library: 1.2.3 (Go)
```

```
Library: 1.2.3
```